### PR TITLE
Update copy on kubernetes/install

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates /kubernetes/install based on the [copydoc](https://docs.google.com/document/d/1oiD6UUomf2daZitS3UOosujYBXPbiOMzPtXVZBq-ph0/edit)

## QA

- Go to 
- Compare against the  [copydoc](https://docs.google.com/document/d/1oiD6UUomf2daZitS3UOosujYBXPbiOMzPtXVZBq-ph0/edit) and resolve comments as you go 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8850

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/kubernetes/install.html [hunk 1]
```diff
@@ -447,10 +493,10 @@ <h2>Read the docs</h2>
           attrs={"class": "p-heading-icon__img"},
           ) | safe
           }}
-          <p class="p-heading-icon__title p-muted-heading">Juju</p>
+          <p class="p-heading-icon__title p-muted-heading">Canonical Kubernetes</p>
         </div>
       </div>
-      <p>Curious to learn about our Charmed Operators and how they automate DevOps and application lifecycle tasks? Browse through the <a href="https://jaas.ai/docs">Charmed OLM documentation</a></p>
+      <p>Curious to learn about Canonical Kubernetes, which will come with 12 years of long term support and built-in features like networking, local storage, dns or ingress? Browse through the <a href="/kubernetes">Canonical Kubernetes documentation</a>.</p>
```
Comment (akbarkz) [inline templates/kubernetes/install.html:499]:
```suggestion
      <p>Curious about Canonical Kubernetes, which will come with 12 years of long term support and built-in features like networking, local storage, dns or ingress? Browse through the <a href="/kubernetes">Canonical Kubernetes documentation</a>.</p>
```

File: templates/kubernetes/install.html [hunk 2]
```diff
@@ -477,14 +523,14 @@ <h2>Kubernetes resources</h2>
           <p class="p-heading-icon__title p-muted-heading" style="color: #fff;">Data Sheet</p>
         </div>
       </div>
-      <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></p>
+      <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise&nbsp;&rsaquo;</a></p>
```
Comment (akbarkz) [inline templates/kubernetes/install.html:526]: This change is not specified in the copy doc

File: templates/kubernetes/install.html [hunk 3]
```diff
@@ -496,7 +542,7 @@ <h2>Kubernetes resources</h2>
           <p class="p-heading-icon__title p-muted-heading" style="color: #fff;">Industry Report</p>
         </div>
       </div>
-      <p><a class="p-link--inverted" href="https://juju.is/cloud-native-kubernetes-usage-report-2021">The Kubernetes and cloud native operations report 2021</a></p>
+      <p><a class="p-link--inverted" href="https://juju.is/cloud-native-kubernetes-usage-report-2022">The Kubernetes and cloud native operations report 2022&nbsp;&rsaquo;</a></p>
```
Comment (akbarkz) [inline templates/kubernetes/install.html:545]:
```suggestion
      <p><a class="p-link--inverted" href="https://juju.is/cloud-native-kubernetes-usage-report-2022">The Kubernetes and cloud native operations report 2022</a></p>
```

**Review by akbarkz — CHANGES_REQUESTED** [review]
You also missed one more change from the copy doc, which is removing version number from "Multi-node, highly available Kubernetes with MicroK8s" section

File: templates/kubernetes/install.html [hunk 4]
```diff
@@ -191,7 +233,7 @@ <h2>How to deploy Kubernetes</h2>
     </table>
     <p><strong>(i).</strong> Use Ubuntu’s platform to run worker nodes on all public clouds (AKS, EKS, and GKE)</p>
     <p>
-      <a class="js-invoke-modal" href="/kubernetes/contact-us?product=kubernetes-install">Looking for help running Kubernetes? Get in touch&nbsp;&rsaquo;</a>
+      <a class="js-invoke-modal" href="/kubernetes/contact-us?product=kubernetes-install">Looking for help running Kubernetes? Get in touch</a>.
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/kubernetes/install.html:236]: Because it is not a link to a different page

File: templates/kubernetes/install.html (hunk 2 above)
Comment (petesfrench) [reply templates/kubernetes/install.html:526]: Chevron usage is not decided on the copydoc level, but on the design level. usually there is some consistency of use within a page, so In this case I was trying to get all the links to align with one another

**Review by akbarkz — APPROVED** [review]

**Pull request merged**